src/components/QuestCard/index.module.scss

```diff
@@ -109,9 +108,16 @@ $border: 2px var(--color-neutral-600) solid;
 }
 
 .currencyName {
-  color: var(--color-neutral-300);
-  font-size: var(--text-md);
+  color: var(--text-weak);
   padding-top: var(--space-3xs);
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
hmm wondering if we should add a tooltip on hover here so users have a way to still see the full reward name? cc @pimentel08

Reply:
same question for `.description`